src/webui/api/torrentscontroller.h

```diff
@@ -30,6 +30,8 @@
 
 #include "apicontroller.h"
 
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
Forward declare it instead: `class BitTorrent::TorrentHandle;` and revert this and also in cpp file.